static/audit.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Audit report</title>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+    </head>
+    <body>
+        <div class="container">
+            <h1>Audit report</h1>
+            <!-- params -->
+            <div class="mb-3">
+                <label for="etherscanApiKey" class="form-label">Etherscan API key</label>
+                <input type="text" class="form-control" id="etherscanApiKey">
+            </div>
+            <div class="mb-3">
+                <label for="rpcUrl" class="form-label">RPC URL</label>
+                <input type="text" class="form-control" id="rpcUrl">
+            </div>
+            <div class="mb-3">
+                <label for="botWalletAddress" class="form-label">Bot wallet address</label>
+                <input type="text" class="form-control" id="botWalletAddress">
+            </div>
+            <div class="mb-3">
+                <label for="githubPat" class="form-label">Github personal access token (with "repo" scope)</label>
+                <input type="text" class="form-control" id="githubPat">
+            </div>
+            <div class="mb-3">
+                <label for="ownerName" class="form-label">Owner name</label>
+                <input type="text" class="form-control" id="ownerName">
+            </div>
+            <div class="mb-3">
+                <label for="repoName" class="form-label">Repository name</label>
+                <input type="text" class="form-control" id="repoName">
+            </div>
+            <button type="button" class="btn btn-primary mb-3" id="getReport">Get report</button>
+            <!-- loader -->
+            <div id="loader" class="mb-3" style="display: none;">
+                Loading...
+                <p id="log"></p>
+            </div>
+            <!-- result table -->
+            <table id="resultTable" class="table" style="display: none;">
+                <thead>
+                  <tr>
+                    <th scope="col">Issue URL</th>
+                    <th scope="col">Transaction URL</th>
+                    <th scope="col">Amount (DAI)</th>
+                  </tr>
+                </thead>
+                <tbody>
+                    <!-- table content -->
+                </tbody>
+            </table>
+        </div>
+
+        <script src="https://cdn.ethers.io/lib/ethers-5.2.umd.min.js" type="application/javascript"></script>
+
+        <script type="module">
```

Comment:
you should use typescript + bundler

Reply:
Yes, this is more of a concept, if it is fine then we can move to TS and a whole lot of other feature